app-libs/stf/src/stf_sgx.rs

```diff
@@ -97,50 +94,37 @@ impl Stf {
 	pub fn get_state(ext: &mut impl SgxExternalitiesTrait, getter: Getter) -> Option<Vec<u8>> {
```

Comment:
I'm not sure if we need this Option now. But since this would be a breaking change, I'll leave as is for now. But might be a future issue to take a look at this.